wv2util/OverlayWindow.xaml.cs

```diff
@@ -0,0 +1,180 @@
+﻿﻿using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Linq;
+using System.Runtime.InteropServices;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+using System.Timers;
+using System.Windows;
+using System.Windows.Controls;
+using System.Windows.Data;
+using System.Windows.Documents;
+using System.Windows.Input;
+using System.Windows.Interop;
+using System.Windows.Media;
+using System.Windows.Media.Imaging;
+using System.Windows.Shapes;
+using Timer = System.Timers.Timer;
+
+
+namespace wv2util
+{
+    /// <summary>
+    /// Interaction logic for Overlay.xaml
+    /// </summary>
+    public partial class OverlayWindow : Window
+    {
+        // The HWND of the target window over which we will render our overlay
+        private IntPtr m_targetHwnd;
+        private IntPtr m_thisHwnd;
+        // The timer that tracks when to check for target HWND updates
+        private Timer m_checkForUpdatesTimer;
+        private Rect m_previousRect;
+        public bool IsClosed { get; private set; } = false;
+        
+        public static OverlayWindow OpenOverlayForHwnd(IntPtr targetHwnd)
+        {
+            var overlayWindow = new OverlayWindow(targetHwnd);
+            if (!overlayWindow.IsClosed)
+            {
+                overlayWindow.Show();
+            }
+            return overlayWindow;
+        }
+        
+        protected OverlayWindow(IntPtr targetHwnd)
+        {
+            this.ShowActivated = false;
+            m_targetHwnd = targetHwnd;
+            Loaded += this.Window_Loaded;
+            InitializeComponent();
+            UpdateOverlayToMatchTarget();
+            this.ShowActivated = false;
+
+            m_checkForUpdatesTimer = new Timer(500);
```

Comment:
In the PR video it looks like the highlight disappears quickly after clicking on the host app entry. We probably want to leave the overlay up as long as the host app entry is selected - or at least have it last a while longer before fading out.